src/slidedown/TaggingContainer.ts

```diff
@@ -0,0 +1,103 @@
+import { TagCategory, TagsObjectWithBoolean } from '../models/Tags';
+import {
+    addDomElement,
+    removeDomElement,
+    addCssClass,
+    removeCssClass,
+    getDomElementOrStub
+} from '../utils';
+import { getLoadingIndicatorWithColor } from './LoadingIndicator';
+import TagUtils from '../utils/TagUtils';
+import { SlidedownCssIds, TaggingContainerCssClasses, TaggingContainerCssIds } from './constants';
+
+export default class TaggingContainer {
```

Comment:
`TaggingContainer` this class should not be in this commit

Reply:
Nice catch. Not sure how that happened. Looks like it is in 8538d073e5acc5903811d499067c6adee9d19ee9

Reply:
Done